### PR TITLE
update model-genomes and compartment-names

### PR DESCRIPTION
**Pull request by cnorsig**

### CODE REVIEW

File: model-genome.txt
Comment (zakandrewking) [inline model-genome.txt:87]: Is there a new iCHO model? What's this line for?

**Review by zakandrewking — CHANGES_REQUESTED** [review]
You'll also need to commit the new model files (JSON & SBML files) to the `models/` directory so I have access to them when loading the DB. Let me know if I can help with that.

File: model-genome.txt
Comment (cnorsig) [reply model-genome.txt:87]: Yeah a new one for DG44 that reflects the loss of de novo purine synthesis so the additional req can enable selection of recomb clones

**Comment by cnorsig** [comment]
[models.zip](https://github.com/SBRG/bigg_models_data/files/3535895/models.zip)

**Comment by zakandrewking** [comment]
also need PMIDs for these

we can merge this after you merge https://github.com/cnorsig/bigg_models_data/pull/2

**Comment by cnorsig** [comment]
anything that has a PMID should have that in the file

**Pull request closed without merging**
